src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.*;
+
+public class Tagging {
+
+	private char _addOrRemoveSign;
+	private final Person _person;
+	private final Tag _tag;
+	
+	
+	public Tagging(char addOrRemoveSign, Person person, Tag tag)
+	{
+		_addOrRemoveSign = addOrRemoveSign;
+		_person = person;
+		_tag = tag;
+	}
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)